src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj

```diff
@@ -20,11 +20,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore" Version="2.32.0" />
-    <PackageReference Include="Grpc.Core" Version="2.32.0" />
-    <PackageReference Include="Grpc.Tools" Version="2.32.0">
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
+    <PackageReference Include="Grpc.AspNetCore" Version="2.39.0" />
```

Comment:
`Grpc.Tools` is brought in automatically with `AspNetCore` -- and doesn't look like the `<PrivateAssets>All</PrivateAssets>`